### DISCUSSION TITLE
Improve Oracle Connector

### DISCUSSION THREAD

**Discussion by nikolaiblackie** [Azure/logicapps#675] [Feature Ideas/Requests]
Would be great to resolve two major known issues. https://learn.microsoft.com/en-us/connectors/oracle/#known-issues-and-limitations

- OUT parameters
- Oracle Functions

These limitations cause us a lot of problems on client projects especially when migrating solutions to Azure where clients do not have VNET connectivity to on-prem and using data gateway.

**Comment by georgeyucao** [comment]
Please resolve output issues for SP and functions (which can return results but not supported). It is a big setback for us as we had to seek alternative ways to track transactions done via SP call as it didn't return outputs for auto-generated entity ids
